lib/components/room-bridge-store.js

```diff
@@ -351,6 +351,20 @@ RoomBridgeStore.prototype.removeEntriesByRemoteRoomData = function(data) {
     return this.delete(data);
 };
 
+/**
+ * Remove entries with this remote room id.
+ * @param {Object} remoteId The remote id.
+ * @return {Promise}
+ * @example
+ *  new RemoteRoom("foobar");
```

Comment:
Maybe one less space before 'new' (but +1 for `@example`, I didn't even know that was a thing)